marlowe-playground-client/src/Simulator/State.purs

```diff
@@ -489,11 +486,7 @@ startSimulation initialTime contract =
                 [ Just $ StartEvent initialTime
                 , case contract of
                     Term Close _ -> Just $ CloseEvent
-                      -- TODO: SCP-3887 unify time construct
-                      ( TimeInterval
-                          (POSIXTime initialTime)
-                          (POSIXTime initialTime)
-                      )
+                      (TimeInterval initialTime initialTime)
```

Comment:
I'm pleased how much of this PR is just removing newtype constructor applications :smile: